src/main/java/org/takes/misc/Href.java

```diff
@@ -182,11 +182,17 @@ public String path() {
      * @return New HREF
      */
     public Href path(final String suffix) {
+        final String slash = "/";
+        final StringBuilder sbr = new StringBuilder(this.uri.toString());
```

Comment:
I think it would be more elegant to do like this:

```
return new Href(
  URI.create(
    new StringBuilder(this.uri.toString().replaceAll("/$", ""))
      .append('/')
      .append(Href.encode(suffix)).toString()
  ),
  URI.create(sbr.toString()),
  this.params
  );
```

don't you think?